src/main/java/io/jenkins/plugins/orka/helpers/Utils.java

```diff
@@ -2,8 +2,15 @@
 
 import java.util.Date;
 
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
 public class Utils {
     public static String getTimestamp() {
         return String.format("[%1$tD %1$tT]", new Date());
     }
+
+    public static String getAsString(Object obj) {
+        return ReflectionToStringBuilder.toString(obj, ToStringStyle.MULTI_LINE_STYLE);
```

Comment:
We have complete control over all of the objects we use `getAsString` with.  
That is - we do not need reflection at all, we can simply either:

- Override the `toString` method of those objects
or 
- Use their property getters to display the information we want

Reply:
At the moment. we are stringifying our own objects, but this may change in the future.
Also, inside the overridden `toString` we need to manually pick the fields that we need to log.

I think we should keep this `getAsString()` method.

Reply:
I'd like to avoid using reflection unless absolutely necessary. What is the problem of manually picking the fields we need to log? It is pretty easy to do with a code generation tool.